test_scripts/Resumption/Handling_errors_from_HMI/012_unsubscribe_from_the_same_vehicledata_without_request_to_hmi_by_error_in_sub_resultCode_during_resumption.lua

```diff
@@ -0,0 +1,138 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. App1 is subscribed to data_1, data_2 and data_3
+-- 2. App2 is subscribed to data_3 and data_4
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App1 and app2 re-register with actual HashId
+-- SDL does:
+--  - start resumption process for both apps
+--  - send VI.SubscribeVehicleData(data_1, data_2, data_3) request related to app1 to HMI
+--  - not send VI.SubscribeVehicleData(data_3, data_4) request related to app2 to HMI
+-- 5. HMI responds with <erroneous> internal resultCode for data_1 to VI.SubscribeVehicleData(data_1, data_2, data_3) request
+-- SDL does:
+--  - process response from HMI
+--  - not restore subscription for app1 (data_1, data_2, data_3)
+--  - respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application app1
+--  - send VI.UnsubscribeVehicleData(data_2) request related to app1 to HMI
+--  - send VI.SubscribeVehicleData(data_4) request related to app2 to HMI
+-- 6. HMI responds with <successful> resultCode to VI.SubscribeVehicleData(data_4) request
+-- SDL does:
+--  - process response from HMI
+--  - restore subscriptions for app2 (data_3, data_4)
```

Comment:
Is data_3 resumed also? Comments above only mention data_4 for app2.

Reply:
Yes, subscription for `data_3` is still active since it's shared data. Please see explanation in [2441#discussion_r478776989](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2441#discussion_r478776989)